src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -136,7 +140,7 @@ public function configureModuleAction($module_name)
         $moduleAccessedId = (int) $moduleAccessed->database->get('id');
 
         // Save history for this module
-        $moduleHistory = $this->getDoctrine()
+        $moduleHistory = $this->entityManager
```

Comment:
Can you please add a `getDoctrine` method in the `FrameworkBundleAdminController`, still the same principle to keep some backward compatibility via this class (but only via this class so that it's easier to clean everything in the next major)